migrations/multitenant/0001-initialmigration.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS storage.tenants (
+  id text PRIMARY KEY,
+  config jsonb NOT NULL
```

Comment:
Can we have explicit columns here Beng? Will be easier to type etc.  

And for the multitenant db, this can be in the public schema itself right? If you want to create it in a storage schema, where is the schema created?

Reply:
Switched to explicit columns and public schema.